smrf/tests/smrf_test_case_lakes.py

```diff
@@ -45,3 +45,71 @@ def tearDownClass(cls):
         folder = os.path.join(cls.base_config.cfg['output']['out_location'])
         if os.path.exists(folder):
             shutil.rmtree(folder)
+
+    def assert_list(self, list1, list2):
+        """Couldn't get the assert_list to work"""
+
+        self.assertTrue(sorted(list1) == sorted(list2))
+
+    def assert_thread_variables(self):
+
+        self.assert_list(
+            self.smrf.distribute['air_temp'].thread_variables,
+            ['air_temp']
```

Comment:
Instead of defining a new list here, you could use the defined values in the classes:
```
ta.BASE_THREAD_VARIABLES
```

This could ensure that we are testing, what we are defining and enqueuing.

Reply:
Makes sense for some of the easier ones. But this test will fail when a new variable is added which may ensure that we go through this process again to ensure that it's being properly added to the variable list.